src/page-templates/content-page/content-page-template.tsx

```diff
@@ -364,81 +375,186 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
-                                    )
-                                }
-                            >
-                                {codeType.toUpperCase()}
-                            </SecondaryTag>
-                        )}
-                        {getSocialButtons(true)}
-                    </div>
-                </div>
-                <div sx={styles.section}>
-                    <div sx={styles.image}>
-                        <Image
-                            alt={parseUndefinedValue(image?.alt)}
-                            src={getPlaceHolderImage(image?.url)}
-                            sx={{
-                                borderRadius: 'inc30',
-                                objectFit: 'cover',
-                            }}
-                            layout="fill"
-                        />
-                    </div>
-                    {!previewMode && ratingSection}
+                        </div>
+                    )}
+                    {!previewMode && !isIndustryEvent && ratingSection}
                     {isCodeExample &&
                         (githubUrl || liveSiteUrl) &&
                         renderExternalExamples({
                             marginTop: 'inc50',
                             marginBottom: ['', null, null, '-inc40'], // negates marginTop from contentBody since externalExamples might not always be present
                         })}
+                    {isIndustryEvent && (
+                        <EventWidget
+                            dates={contentDate}
+                            location={location}
+                            virtualLink={virtualLink}
+                            virtualLinkText={virtualLinkText}
+                            registrationLink={registrationLink}
+                            buttonStyles={{
+                                marginTop: '-inc30', // negates marginTop from the component's internal styles
+                                marginBottom: '-inc20', // negate spacing from content body
+                                width: ['100%', null, null, 'auto'],
+                            }}
+                            wrapperStyles={{
+                                display: ['block', null, null, 'none'],
+                                marginTop: 'inc40',
+                            }}
+                        />
+                    )}
                 </div>
                 <div sx={styles.bodySection}>
                     <DocumentBody content={contentAst} />
+                    {isIndustryEvent && (
+                        <>
+                            <TypographyScale
+                                variant="heading5"
+                                sx={{
+                                    marginTop: 'inc60',
+                                    marginBottom: 'inc40',
+                                }}
+                            >
+                                Speakers
```

Comment:
do we need Header speakers when there are no speakers ?
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/9810710/212196845-e26391ca-01b4-4e57-ac93-9807a54718c2.png">

Reply:
good catch, ill adjust